src/app/forms/templates/test-records/section-templates/customDefects/custom-defects-section.template.ts

```diff
@@ -0,0 +1,50 @@
+import { FormNode, FormNodeTypes, FormNodeViewTypes } from '@forms/services/dynamic-form.types';
+
+export const CustomDefectsSection: FormNode = {
+  name: 'customDefectsSection',
```

Comment:
Not sure where to put this, fairly sure we need prohibition issued as part of custom defects but can't see it in your template?

Reply:
Nevermind, just saw we have a separate ticket for prohibitions further down the list, even though they are technically part of custom defects